livre/eq/Config.cpp

```diff
@@ -363,25 +343,18 @@ bool Config::switchToViewCanvas( const eq::uint128_t& viewID )
     return true;
 }
 
-void Config::handleEvents()
-{
-    eq::Config::handleEvents();
 #ifdef LIVRE_USE_ZEQ
+void Config::handleNetworkEvents()
+{
     _impl->communicator->handleEvents();
     _impl->communicator->publishHeartbeat();
```

Comment:
The GUI should ask for a transfer function repeatedly until it gets one, and then stop.